src/main/java/org/broadinstitute/hellbender/tools/evoquer/GnarlyGenotyperEngine.java

```diff
@@ -0,0 +1,492 @@
+        vcfBuilder.attribute(GATKVCFConstants.STRAND_ODDS_RATIO_KEY, StrandOddsRatio.formattedValue(StrandOddsRatio.calculateSOR(StrandBiasTest.decodeSBBS(SBsum))));
+        annotationDBBuilder.attribute(GATKVCFConstants.SB_TABLE_KEY, SBsum);
+
+        vcfBuilder.genotypes(calledGenotypes);
+        annotationDBBuilder.noGenotypes();
+        vcfBuilder.alleles(targetAlleles);
+
+        for (final Class c : allASAnnotations) {
+            try {
+                InfoFieldAnnotation annotation = (InfoFieldAnnotation) c.newInstance();
+                if (annotation instanceof AS_StandardAnnotation && annotation instanceof ReducibleAnnotation) {
+                    ReducibleAnnotation ann = (ReducibleAnnotation) annotation;
+                    if (variant.hasAttribute(ann.getRawKeyName())) {
+                        if (!stripASAnnotations) {
+                            final Map<String, Object> finalValue = ann.finalizeRawData(vcfBuilder.make(), variant);
+                            finalValue.forEach((key, value) -> vcfBuilder.attribute(key, value));
+                            annotationDBBuilder.attribute(ann.getRawKeyName(), variant.getAttribute(ann.getRawKeyName()));
+                        }
+                    }
+                }
+            }
+            catch (Exception e) {
+                throw new IllegalStateException("Something went wrong: ", e);
+            }
+        }
+        //since AS_FS and AS_SOR share the same raw key, we have to wait to remove raw keys until all the finalized values are added
+        for (final Class c : allASAnnotations) {
+            try {
+                InfoFieldAnnotation annotation = (InfoFieldAnnotation) c.newInstance();
+                if (annotation instanceof AS_StandardAnnotation  && annotation instanceof ReducibleAnnotation) {
+                    ReducibleAnnotation ann = (ReducibleAnnotation) annotation;
+                    if (variant.hasAttribute(ann.getRawKeyName())) {
+                        vcfBuilder.rmAttribute(ann.getRawKeyName());
+                    }
+                }
+            }
+            catch (Exception e) {
+                throw new IllegalStateException("Something went wrong: ", e);
+            }
+        }
+        if (variant.hasAttribute(GATKVCFConstants.AS_VARIANT_DEPTH_KEY)) {
+            vcfBuilder.attribute(GATKVCFConstants.AS_ALT_ALLELE_DEPTH_KEY, AS_QualByDepth.finalizeRawGVCFVarDPValues(
+                    variant.getAttributeAsString(GATKVCFConstants.AS_VARIANT_DEPTH_KEY, null), targetAlleles.size()));
+            //Since this was added by ReblockGVCFs and not the AnnotationEngine, I feel justified in taking it out without using the AnnotationEngine
+            vcfBuilder.rmAttribute(GATKVCFConstants.AS_VARIANT_DEPTH_KEY);
+        }
+
+        annotationDBBuilder.alleles(targetAlleles);
```

Comment:
Add comment that we don't return `annotationDBBuilder.make()` because we're assuming the caller is re-using the same instance across calls.